src/lib/model/InboundTransfersModel.js

```diff
@@ -65,24 +66,34 @@ class InboundTransfersModel {
      * Queries the backend API for the specified party and makes a callback to the originator with our dfspId if found
      */
     async getParticipantsByTypeAndId(idType, idValue, sourceFspId) {
+        const histTimerEnd = Metrics.getHistogram(
+            'inbound_get_participants_type_id',
```

Comment:
Same comment applies here for metric re-use based on map, and for the rest of the file.

Reply:
decided not to go with this option as name in route map requires the path to access